build.gradle.kts

```diff
@@ -237,7 +237,9 @@ subprojects {
                     inputs.property("moduleName", moduleName)
 
                     manifest {
-                        attributes("Automatic-Module-Name" to moduleName)
+                        attributes(
+                                "Automatic-Module-Name" to moduleName,
+                                "Implementation-Version" to "${project.version}")
```

Comment:
Can we add all the attributes from https://github.com/open-telemetry/opentelemetry-java/blob/v0.14.1/build.gradle#L151-L156 back? Especially implementation-title.

Reply:
Certainly! I initially wanted to close the gap mentioned in #3032 but rest of the attributes makes sense as well.